src/Components/Cart.jsx

```diff
@@ -0,0 +1,71 @@
+import { cartContext } from "../Context/CartProvide"; 
+import { useContext} from "react";
+import Swal from 'sweetalert2'
+import Checkout from "./Checkout";
+
+function Cart(){
+    let {cart} = useContext(cartContext)
+    let {setCart} = useContext(cartContext)
```

Comment:
use const instead of let, since these are immutable
```suggestion
    const {cart,setCart} = useContext(cartContext)
```